framework/output/RunOutputGroup.kt

```diff
@@ -96,16 +97,13 @@ internal class RunOutputGroup constructor(
     }
 
     private fun publishQueryResponseTime() = runner.startTime?.let { startTime ->
-        val duration = (runner.endTime ?: System.currentTimeMillis()) - startTime
-        Service.status.publish(
-            QUERY_RESPONSE_TIME,
-            Duration.milliseconds(duration).toString()
-        )
+            val duration = (runner.endTime ?: System.currentTimeMillis()) - startTime
+            Service.status.publish(QUERY_RESPONSE_TIME, duration.milliseconds.toString())
     } ?: Unit
 
     private fun publishOutputResponseTime() = runner.endTime?.let { queryEndTime ->
-        val duration = (endTime ?: System.currentTimeMillis()) - queryEndTime
-        Service.status.publish(OUTPUT_RESPONSE_TIME, Duration.milliseconds(duration).toString())
+            val duration = (endTime ?: System.currentTimeMillis()) - queryEndTime
+            Service.status.publish(OUTPUT_RESPONSE_TIME, duration.milliseconds.toString())
```

Comment:
Let's refactor these two `duration` variables as well (here and L100) - how about renaming them (both) to `responseTime`?

Reply:
Done.